sale_blanket_order/wizard/create_sale_orders.py

```diff
@@ -64,10 +64,9 @@ def _default_lines(self):
         self._check_valid_blanket_order_line(bo_lines)
 
         lines = [
-            (
-                0,
-                0,
+            fields.Command.create(
                 {
+                    "analytic_distribution": bol.analytic_distribution,
```

Comment:
@StefanRijnhart Does the `_compute_analytic_distribution`miss in `sale.blanket.order.line` model ? Either, this will be void, no ?

Reply:
Yes, you are right! All the values here for related fields are void. I removed them.

Reply:
@rousseldenis Is it OK for you now?